docs/canvas/canvas-getting-started.asciidoc

```diff
@@ -2,31 +2,158 @@
+
+TIP: Want to delete an element? Click the back button on your browser.
+
+[float]
+=== Show off your data
+
+To show what your data can do, add charts, graphs, progress monitors, and more.
+
+. Click *Add element*, then click *Area chart*. 
+
+. To connect the element to your own data source, make sure that the element is selected, then click *Data*.
 
-You’re in! The first thing to do is to make a copy of this 
-workpad so you can come back to it later if needed. We’ll be making a mess 
-of it in this tutorial. 
+.. Click *Change your data source*, then click *Elasticsearch SQL*.
 
-. Click the name of the workpad in the lower left hand corner. 
+.. To obtain the taxless total price by date, enter `SELECT order_date, taxless_total_price FROM "kibana_sample_data_ecommerce" ORDER BY order_date` into the *Elasticsearch SQL query* field.
 +
-You’ll notice this is similar to the first screen. You could switch to a new 
-workpad from here, but let's not for now.
+Although you used the Elasticsearch SQL data source for the metric and area chart elements, each element can display a different data source. Pages and workpads often contain multiple data sources.
+
+.. To verify that the data is correct, click *Preview*. If you like what you see, click *Save*.
+
+. Specify how to display the data.
+
+.. Click *Display*
+
+.. From the *X-axis* drop-down lists, select *Value*, then select *order_date*.
+
+.. From the *Y-axis* drop-down lists, select *Value*, then select *taxless_total_price*.
+
+[role="screenshot"]
+image::images/canvas-area-chart.png[]
+
+[float]
+=== Show how your data changes over time
 
-. Click the *Clone* icon in the *[eCommerce] Revenue Tracking* row.
+To focus your data on a specific time range, apply a time filter. 
+
+. Click *Add element*, then click *Time filter*.
+
+. Specify how to display the data.
+
+.. Click *Display*
+
+.. To use the date time field from the sample data, enter `order_date` in the *Column* field, then click *Set*.
 +
-You now have a new workpad called *[eCommerce] Revenue Tracking - Copy*.
+To see how the data changes, set the time filter to *Last 7 days*. As you change the time filter options, the metrics dynamically update.
+
+[role="screenshot"]
+image::images/canvas-time-filter.png[]
+
+Your workpad is now complete! Use the icons in the upper left corner to:
+
+* Display your workpad in fullscreen mode
+
+* Download your workpad as a PDF report or JSON object
+
+* Configure the refresh rate for your data
+
+* Share your workpad
+
+[float]
+=== Next steps
+Now that you know the Canvas basics, you're ready to explore on your own. 
+
+Here are some things to try:
+
+* Play with the {kibana-ref}/add-sample-data.html[sample Canvas workpads]. 
+
+//* Learn more about:
+
+//** {kibana-ref}/canvas-workpad[Workpads]
+//** Assets
+//** Data sources
+//** Elements
 
+* Deep dive into the {kibana-ref}/canvas-function-reference.html[expression language and functions] that drive Canvas.
```

Comment:
You'll need to remove the next section, "Using the workpad"

Reply:
Since I'm just going to edit the existing file, I'm going to keep this link here.